.gitignore

```diff
@@ -7,18 +7,40 @@
 .pydevproject
```

Comment:
(Can't comment above, as it's empty)

Could you add a small bit of descriptive text inside `.KALITE_SOURCE_DIR` to enable opportunistic learning about its purpose?